content/en/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -64,9 +64,10 @@ If your component repository is **not** in this organization:
 
 1. Invite `openshift-ci-robot` and `openshift-merge-robot` into your organization or add them as collaborators for the repository.
 1. Contact a CI administrator to accept the invitation
-1. By default, we enable [branch-protection](TODO: Add link once https://github.com/openshift/ci-docs/pull/7 merges) for
+1. By default, we enable [branch-protection](../architecture/branch-protection.md) for
```

Comment:
This doesn't work, needs to be `/docs/architecture/branch-protection`

Reply:
Updated.